### PR TITLE
Some new translated string

### PR DESCRIPTION
**Pull request by domenc**
issue #137

### CODE REVIEW

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -330,3 +304,22 @@ msgstr "Zadnja prijava:"
...
+
+#: .\panels\onlineusers\panel.py:9
+msgid "Online users"
+msgstr "Povezani uporabniki"
+
+#: .\panels\onlineusers\templates\panel\onlineusers\panel.html.py:26
+msgid "Search users"
+msgstr "Išči uporabnike"
+
+#~ msgid "Show/Hide Users"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po:324]: Just remove unused translations.

File: piplmesh/locale/sl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -330,3 +304,22 @@ msgstr "Zadnja prijava:"
 #: .\frontend\templates\user\user.html.py:25
 msgid "Facebook link"
 msgstr "Facebook povezava"
+
+#: .\panels\dummy\panel.py:11
+msgid "Dummy panel"
+msgstr "Testna plošča"
```
Comment (mitar) [inline piplmesh/locale/sl/LC_MESSAGES/django.po]: I don't like _plošča_. Maybe _ploščica_?

**Comment by mitar** [comment]
Please update from main repository.

**Comment by domenc** [comment]
Fix it in new pull request. Closing this one

**Pull request closed without merging**